VictoriaAlbert/VictoriaAlbert/APIRequestManager.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  APIRequestManager.swift
+//  VictoriaAlbert
+//
+//  Created by Thinley Dorjee on 11/10/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import Foundation
+
+class APIRequestManager{
+    static let manager = APIRequestManager()
+    private init(){}
+    
+    func getData(endPoint: String, completion: @escaping ((Data?)->Void)){
+        guard let myURL = URL(string: endPoint) else {return}
+        let session = URLSession(configuration: URLSessionConfiguration.default)
+        session.dataTask(with: myURL) { (data: Data?, response: URLResponse?, error: Error?) in
+            if error != nil{
+                print("Error encounter \(error)")
+            }
+            guard let validData = data else {return }
+            completion(validData)
+            }.resume()
+    }
+    
+    
+    func downloadImage(urlString: String, callback: @escaping (Data) -> () ) {
```

Comment:
If you look closely you'll see ```downloadImage``` is the same as ```getData``` so you don't need it. You can get JSON and images with ```getData```.